src/main/webapp/view/transcript/transcript-seeker.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Seeks to a line in the transcript. */
+export default class TranscriptSeeker {
+  static #TRANSCRIPT_CONTAINER = 'transcript-lines-container';
+
+  #currentTranscriptLine;
+  #eventController;
+
+  /**
+   * Creates an instance of `TranscriptSeeker` for loading
+   * transcript lines onto the DOM.
+   *
+   * @param eventController An event controller instance that
+   *    will help relay the current time to other objects.
+   */
+  constructor(eventController) {
+    this.#eventController = eventController;
+    // TODO: Add the event listeners.
+  }
+
+  /**
+   * Returns the `currentTranscriptLine` if it exists. Else, returns
+   * undefined.
+   */
+  currentTranscriptLine() {
+    if (this.#currentTranscriptLine == null) {
+      // If the first transcript line doesn't exist, currentTranscript is
+      // assigned to be undefined.
+      this.#currentTranscriptLine =
+          document.getElementsByTagName('transcript-line')[0];
+    }
+    return this.#currentTranscriptLine;
+  }
+
+  /**
+   * Scrolls `transcriptLine` to the top of the transcript area.
```

Comment:
This confuses me, how can we scroll 1 transcript line to the top of the transcript area. Do you mean you have to transcriptArea start with transcriptLine?

Reply:
Yes, that is what I meant. I updated it so hopefully, it is more clear.